cirq/ops/raw_types.py

```diff
@@ -199,6 +214,21 @@ def transform_qubits(self: TSelf_Operation,
         """
         return self.with_qubits(*(func(q) for q in self.qubits))
 
+    def __control__(self,
```

Comment:
`_control_` instead of `__control__`. Double-underscore is reserve by python.

What do you think of using the name `controlled_by` instead of `_control_`? E.g. `cirq.X(a).controlled_by(b)`

Reply:
Renamed to controlled_by